partner_email_unique/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Partner unique email",
+    "summary": "Add an unique constraint to email field",
+    "version": "9.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "https://odoo-community.org/",
```

Comment:
I missed this one on the review, sorry: website should be: ```https://github.com/oca/partner-contact```

Reply:
👌  No worries